docs/dist/js/bootstrap.min.js

```diff
@@ -1,5 +1,5 @@
 /*!
- * Bootstrap v5.13.1 (http://bootstrap.transferwise.com)
+ * Bootstrap v5.13.0 (http://bootstrap.transferwise.com)
```

Comment:
Interesting that the version is changing from `5.13.1` to `5.13.0` here?

Reply:
Yea, I just had 1 file where it went from 5.13.6 to 5.13.1  (where 5.13.1) was the latest 😅

Reply:
Ugh